test.md

```diff
@@ -0,0 +1 @@
+this file should be ignored by git 
```

Comment:
- [ ] Please Make sure that the last file(test.md file) is actually ignored by git and is not present in your GitHub repository. you can use 
`# ignore Test.md file`
 ` test.md` in your gitignore file